### ISSUE TITLE
Coutdown

### ISSUE THREAD

**Issue by rolandverner** [uikit/uikit#3468]
The problem is that countdown takes time from the computer as a basis and not from the server, and it turns out that the countdown timer shows different time for different regions of the planet whose time is different

### UIkit version
3.0.0-rc.8

### Reproduction link
https://codepen.io/roland-verner/pen/eKqzdL

### Browser
All Browsers

### Steps to reproduce
Just change your time on the computer and you'll see that the countdown will also change its time

**Comment by peterbrinck** [comment]
It's not that easy to overcome, since that's how JavaScript works with time.
It uses the browsers time.

A workaround would be to do some ajax and call a server, probably some NTP service, and then use the returned date and time to calculate with.

**Comment by RolandVerner** [comment]
yes, but one solution is to add another time parser that will take the current time. 
Example:
`<div uk-countdown="datenow: 2018-07-14T14:01:44+00:00; date: 2018-07-14T14:01:44+00:00">`
Or make it take time from the server itself.

**Comment by janschoenherr** [comment]
Yes, the countdown depends on the user's time. As far as I know, it does not depend on the user's timezone, if you provide a timezone in your date string. `2018-07-14T14:01:44+0000`

**Comment by jamesclavel** [comment]
Hi there, new to using UIkit's Countdown feature. does any one encounter the countdown timer continue counting down (_after being stopped:_ `UIkit.countdown(element).stop()`) when you click on other browser tab(s).

looking forward for a solution for the countdown timer not to resume unless a "resume" button is clicked. thanks.

**Comment by jamesclavel** [comment]
> Hi there, new to using UIkit's Countdown feature. does any one encounter the countdown timer continue counting down (_after being stopped:_ `UIkit.countdown(element).stop()`) when you click on other browser tab(s).
> 
> looking forward for a solution for the countdown timer not to resume unless a "resume" button is clicked. thanks.
> 
> a sample codepen code: https://codepen.io/jamesclavel/pen/arjLxv
> 
> when the codepen window is loaded - it should stop the countdown. then you can try and open a new tab or click the other opened tab in the browser and you can see the countdown timer resumes.

Just in case somebody stumble upon this comment, I resolved the issue (perhaps for what I needed) by removing the `events` block from the countdown.js component.
[https://github.com/uikit/uikit/blob/develop/src/js/components/countdown.js](https://github.com/uikit/uikit/blob/develop/src/js/components/countdown.js)

**Comment by RolandVerner** [comment]
> > Hi there, new to using UIkit's Countdown feature. does any one encounter the countdown timer continue counting down (_after being stopped:_ `UIkit.countdown(element).stop()`) when you click on other browser tab(s).
> > looking forward for a solution for the countdown timer not to resume unless a "resume" button is clicked. thanks.
> > a sample codepen code: https://codepen.io/jamesclavel/pen/arjLxv
> > when the codepen window is loaded - it should stop the countdown. then you can try and open a new tab or click the other opened tab in the browser and you can see the countdown timer resumes.
> 
> Just in case somebody stumble upon this comment, I resolved the issue (perhaps for what I needed) by removing the `events` block from the countdown.js component.
> https://github.com/uikit/uikit/blob/develop/src/js/components/countdown.js

Ok, but it did not solve the problem the countdown takes the browser time as the current time, and I need to set server time as time from countdown starts
